### PR TITLE
Add Phoenix parachain

### PR DESCRIPTION
**Pull request by yuyunhong**

### CODE REVIEW

**Comment by yuyunhong** [comment]
Add Phoenix parachain

File: packages/apps/public/locales/en/apps-config.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
   "rpc.nodle-arcadia": "Arcadia",
   "rpc.nodle-main": "Nodle",
   "rpc.phala": "Phala PoC-3",
+  "rpc.phoenix": "PHOENIX Mashnet",
```
Comment (jacogr) [inline packages/apps/public/locales/en/apps-config.json:53]: This doesn't quite seem to match up above? All lowercase there, all uppercase here? (Small nitpick, but the i18n generation will just warn, on this one)

Comment (yuyunhong) [reply packages/apps/public/locales/en/apps-config.json]: Thank you. I have fixed the issues and submitted the changes.

**Review by jacogr — APPROVED** [review]

**Comment by jacogr** [comment]
Thank you.

**Pull request merged**